pkg/nsx/services/common/types.go

```diff
@@ -58,6 +66,8 @@ var (
 	ResourceTypeClusterControlPlane = "clustercontrolplane"
 	// ResourceTypePrincipalIdentity is used by NSXServiceAccountController, and it is MP resource type.
 	ResourceTypePrincipalIdentity = "principalidentity"
+	ResourceTypeVPC               = "VPC"
```

Comment:
ResourceTypeVPC should be "Vpc" instead of "VPC"

Reply:
Thanks, this should not be used, removed this as we already have ResourceTypeVpc